tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -36,7 +36,9 @@ provisioningInput is needed to the run the K3S tests, specifically kubernetesVer
     "k3sKubernetesVersion": ["v1.24.4+k3s1"],
     "providers": ["linode", "aws", "azure", "harvester"],
     "nodeProviders": ["ec2"],
-    "hardened": true
+    "hardened": true,
+    "psaEnabled": true,
+    "psact": ["rancher-privileged", "rancher-restricted"]
```

Comment:
Yeah, I just put both values so that testers knew the two options they have. I can refactor that if it makes more sense to do that.

If you leave `psact` empty in the configuration file, the PSACT specific tests proceed when they shouldn't. If you do define `psact`, the test is specifically looking for `rancher-privileged` or `rancher-restricted`. So there has to be a condition set that makes the implementation a bit extra IMO.

I went with the `psaEnabled` option so that even if you have psact defined in your configuration YAML file, it gets ignored if `psaEnabled` is set to false, which by default it is.

Reply:
yeah, having psact as a [] string, when really we are expecting only 1 string value, I think would make sense to refactor.  to `psact string`. 


instead of having `psaEnabled`, why not have `if len(psact string) > 0` instead of `if psaEnabled`? this way, we would cut down to 1 clear variable. Thoughts?

Reply:
That potentially could work, the only thing is if someone puts something other than `rancher-restricted` or `rancher-privileged`, the test will cause a panic and it may not be immediately obvious it's a config issue and not the test.

If we don't think it's that big of a concern, I can make that change.

Reply:
I think I actually found a clean away around my concerns. I'll test it and push a change up.

Reply:
Resolved. I was able to remove `psaEnabled` and keep only `psact` as a string value.